tox.ini

```diff
@@ -45,20 +34,11 @@ commands=codecov -X gcov --file {toxworkdir}/coverage.xml
 
 [testenv:docs]
 extras =
-        docs
-        testing
+	docs
+	testing
 changedir = docs
 commands =
-        {envpython} -m sphinx \
-          -j auto \
-          -b html \
-          --color \
-          -a \
-          -n \
-          -W \
```

Comment:
this change kills strict validation

Reply:
This feature is probably worth retaining (done in a283c8776). I'll consider applying this to all projects (jaraco/skeleton#29).